src/__mocks__/next/config.ts

```diff
@@ -0,0 +1,3 @@
+export default () => ({
```

Comment:
`ASK`
Why you name this directory as `__mocks__` ?

Reply:
`__mocks__` dir will be automatically mocked in Jest without configuration.
https://jestjs.io/docs/ja/manual-mocks

Reply:
And I referred to the following
https://spectrum.chat/next-js/general/how-to-setup-jest-tests-to-use-publicruntimeconfig~27f8cee1-d4a4-4b68-bb62-ea6562a77544

`publicRuntimeConfig` is a setting for the entire project. 
so I configured in __mocks__ dir.